test/ReadsUtils_server_test.py

```diff
@@ -1050,22 +1100,41 @@ def test_upload_fail_interleaved_for_single(self):
              'fwd_id': ret['id'],
              'name': 'bar'
              },
-            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret['id'])
+            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved.fastq')
         self.delete_shock_node(ret['id'])
 
-    def test_upload_fail_interleaved_for_paired(self):
-        ret1 = self.upload_file_to_shock('data/Sample1.fastq')
-        ret2 = self.upload_file_to_shock('data/Sample5_interleaved.fastq')
-        self.fail_upload_reads(
-            {'sequencing_tech': 'tech',
-             'wsname': self.ws_info[1],
-             'fwd_id': ret1['id'],
-             'rev_id': ret2['id'],
-             'name': 'bar'
-             },
-            'Invalid fasta file /kb/module/work/tmp/rev/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret2['id'])
+    def test_bad_paired_end_reads(self):
+        ret1 = self.upload_file_to_shock('data/small.forward.fq')
+        ret2 = self.upload_file_to_shock('data/Sample5_noninterleaved.1.fastq')
+        self.fail_upload_reads({'fwd_id': ret1['id'],
+                                'rev_id': ret2['id'],
+                                'sequencing_tech': 'seqtech-pr1',
+                                'wsname': self.ws_info[1],
+                                'name': 'pairedreads1',
+                                'interleaved': 0},
+                               'Interleave failed - reads files do not have ' +
+                               'an equal number of records. forward Shock node ' +
+                               '/kb/module/work/tmp/fwd/small.forward.fq, ' +
+                               'filename small.forward.fq, reverse Shock node ' +
+                               '/kb/module/work/tmp/rev/Sample5_noninterleaved.1.fastq, ' +
+                               'filename Sample5_noninterleaved.1.fastq')
```

Comment:
This error message doesn't make sense - it's saying shock nodes but giving file names. It needs to provide shock nodes to make debugging possible.